io_export_cryblend/utils.py

```diff
@@ -933,6 +933,9 @@ def apply_animation_scale(armature):
 
 def get_bone_geometry(bone_name):
     try:
+        if bone_name.endswith("_Phys"):
+            bone_name = bone_name[:-5]

```

Comment:
Ah, I almost forgot, strings are immutable in Python, so they cannot be changed directly.  You must make a copy.  So your original code is fine.  My bad.